tests/Microsoft.Data.Analysis.Tests/Microsoft.Data.Analysis.Tests.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFrameworks>net461;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp3.0</TargetFrameworks>
```

Comment:
Just so I understand, on a Mac, this would define both net461 and netcoreapp3.0 in line 4 and then be overwritten to just netcoreapp3.0 in line 5?

Reply:
Correct, that's how MSBuild properties get interpreted - top to bottom. Doing it this way works around some VS bugs that inspect the TargetFrameworks node:

Example:
https://github.com/dotnet/project-system/issues/4149